manifests/src/upload_index.sh

```diff
@@ -1,20 +1,59 @@
 # Find the latest version of the dataset
-ZENODO_ENDPOINT="https://zenodo.org"
-DEPOSITION_PREFIX="${ZENODO_ENDPOINT}/api/deposit/depositions"
 ORIGINAL_ID="13892061"
```

Comment:
If we want to make it fully reusablae we need to also pass the ID of the root record (the first version uploaded) as an argument.